src/FakeItEasy.Analyzer/DiagnosticDefinitions.cs

```diff
@@ -0,0 +1,50 @@
+namespace FakeItEasy.Analyzer
+{
+    using System;
+    using System.Collections.Immutable;
+    using System.Reflection;
+    using Microsoft.CodeAnalysis;
+
+    public static class DiagnosticDefinitions

```

Comment:
Should really be internal, but it's used in the unit tests. The unit test assembly can be added as a friend assembly, but it needs to be strong-named for that. Since it's just a prototype at this point, I didn't do it yet.